templates/rhpam78-prod.yaml

```diff
@@ -499,6 +499,38 @@ parameters:
   description: Whether to add to the current roles, or replace the current roles with the mapped ones. Replaces if set to true.
   name: AUTH_ROLE_MAPPER_REPLACE_ROLE
   required: false
+## AMQ Begin
```

Comment:
Please open for prod template change a new PR under issue RHPAM-2919

Reply:
@jakubschwan here https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/434

Reply:
@jakubschwan done